Updating the documentation for Configuring SAML & SCIM in OKTA to include steps for adding the GHEC - Org app integration.

adamrr724 opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:

Closes : https://github.com/github/docs/issues/22475

<!-- If there's an existing issue for your change, please replace ISSUE above with a link to the issue.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

I have added steps to the "Configuring SAML single sign-on and SCIM using Okta" page to include the steps for adding the app integration necessary to set up SAML on OKTA using GitHub. This includes a reusable widget for searching for and adding the correct app integration and includes one screenshot.

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

adamrr724 committed:
Update the OKTA Configuration Instructions to include steps to add/enable GitHub OAuth Integration.

adamrr724 committed:
Create okta-add-ghec-org-integration.md

adamrr724 committed:
Add the resuable widget for searching for and adding the GHEC - org app integration in OKTA.

adamrr724 committed:
Updating the links to remove copy.md

adamrr724 committed:
Merge branch 'main' into main

adamrr724 committed:
Remove the .md file type.

adamrr724 committed:
Make OKTA App Integration reusable widget steps more clear.

adamrr724 committed:
Update Title of Section and better word some of the instructions.

adamrr724 committed:
Merge branch 'main' into main

cmwilson21 commented:
👋 @adamrr724 Thanks for submitting a PR! I edited your post slightly to link it to your issue ✨ 

I'll get this triaged for review! ⚡

lecoursen commented on data/reusables/saml/okta-add-ghec-org-integration.md:
```diff
@@ -0,0 +1,3 @@
+1. Search for the "GitHub Enterprise Cloud - Organization". ![Screen Shot 2022-12-05 at 11 25 01 AM](https://user-images.githubusercontent.com/16549089/205725277-14856ca9-b64d-4b78-b235-86c5fdb5aebc.png)
+1. **Add Integration**. ![Screen Shot 2022-12-05 at 10 27 37 AM](https://user-images.githubusercontent.com/16549089/205713938-4339ceb1-a122-4557-a392-6c34f5a984e1.png)
```
```suggestion
1. Click **Add Integration**. 

   ![Screenshot of the "Add Integration" button](/assets/images/help/saml/add-integration-button-okta.png)
```

lecoursen reviewed (commented):
Thanks @adamrr724! I'm going to apply some suggestions to better follow our style guide and content models. Then, I need you to save your new images in the repository, please... I've updated the image references to include the directory and file names you should use. After you've done that, let me know, and I'll do another review. 😊

lecoursen committed:
Add :nail_care:

IcebergVerse reviewed (commented):
Add 💅

IcebergVerse reviewed (commented):
content/organizations/managing-saml-single-sign-on-for-your-organization/configuring-saml-single-sign-on-and-scim-using-okta.md

adamrr724 committed:
Merge branch 'main' into main

adamrr724 committed:
Add image files to cprrect location.

adamrr724 committed:
Updating the titles for clarity.

Part of the reason these docs needed to be updated was to make these titles/steps more specific. Oauth App Integration in OKTA ≠ GitHub Oauth App, and specifying that this is being set up for SAML -- specifically for a GHEC organization (and not for an enterprise) -- is very important in providing clarity.

adamrr724 commented:
Adding my description from the most recent commit to add context as to why those titles should remain more specific:

Part of the reason these docs needed to be updated was to make these titles/steps more specific. Oauth App Integration in OKTA ≠ GitHub Oauth App, and specifying that this is being set up for SAML -- specifically for a GHEC organization (and not for an enterprise) -- is very important in providing clarity.

lecoursen commented:
>Part of the reason these docs needed to be updated was to make these titles/steps more specific. Oauth App Integration in OKTA ≠ GitHub Oauth App

Thanks for that clarification! 

>and specifying that this is being set up for SAML -- specifically for a GHEC organization (and not for an enterprise) -- is very important in providing clarity.

I agree, but that is true for the entire article, so we should be establishing that in the article title and overall conceptual content for the whole article, not individual section headings. I agree we could do a better job of that (probably because SAML only existed at the organization level when this article was originally created/named), but that's beyond the scope of this update. Could you please open an issue in `github/docs-internal` to track those additional changes internally? Thanks!

lecoursen commented on content/organizations/managing-saml-single-sign-on-for-your-organization/configuring-saml-single-sign-on-and-scim-using-okta.md:
```diff
@@ -31,8 +31,13 @@ After you enable SCIM, the following provisioning features are available for any
 Alternatively, you can configure SAML SSO for an enterprise using Okta. SCIM for enterprise accounts is only available with Enterprise Managed Users. For more information, see "[Configuring SAML single sign-on for your enterprise using Okta](/admin/identity-and-access-management/managing-iam-for-your-enterprise/configuring-saml-single-sign-on-for-your-enterprise-using-okta)" and "[Configuring SCI ...
 
-## Adding the {% data variables.product.prodname_ghe_cloud %} application in Okta
+## Configuring SAML for a {% data variables.product.prodname_ghe_cloud %} organization in Okta
```
Based on your clarification in https://github.com/github/docs/pull/22476#issuecomment-1342959706, I agree that specifying SAML is a good idea. However, we need to keep heading titles as short as possible to increase scanability and reduce the cognitive load of reading the article. As I mentioned in https://github.com/github/docs/pull/22476#issuecomment-1343011987, it would be great if you could open an issue to internally discuss other ways how we clarify that this entire article relates to organizations only. 🙇🏻‍♀️ 

```suggestion
## Configuring SAML in Okta
```

lecoursen committed:
Shorten heading

lecoursen committed:
Merge branch 'main' into main

mahn4889 reviewed (commented):
I am with you
